rust/src/lib.rs

```diff
@@ -1846,6 +1846,8 @@ pub unsafe extern "C" fn zcashlc_latest_cached_block_height(
 /// - The memory referenced by `tx` must not be mutated for the duration of the function call.
 /// - The total size `tx_len` must be no larger than `isize::MAX`. See the safety
 ///   documentation of pointer::offset.
+/// - `txid_ret` must be non-null and must point to an allocated 32-byte region of memory. On
+///   successful execution this will contain the txid of the decrypted transaction.
```

Comment:
```suggestion
/// - `txid_ret` must be non-null and valid for writes of 32 bytes with an alignment of 1.
///   On successful execution (return value of 1) this will contain the txid of the decrypted
///   transaction.
```
At line 1836 add:
```rust
/// Returns 1 on success or -1 on failure.
///
```